src/grecon/grecon.c

```diff
@@ -103,7 +103,7 @@ void grecon(struct grecon_conf* param,  const long dims1[DIMS], complex float* o
 		sens1 = cov1;
 	}
 
-	fftmod(DIMS, dims1, FFT_FLAGS, sens1, sens1);
```

Comment:
I think this is wrong. If caltwo is run here cov1_dims would be smaller than dims1 which is the correct size for the sensitivities.